stories/stories.scss

```diff
@@ -46,3 +46,6 @@ $fs-default: $fs-value * $fs-scale;
 $padding-vertical: $PADDING 0;
 $padding: $PADDING $PADDING * 2;
 
+body {
+    font-family: 'Fira Sans', sans-serif !important;
```

Comment:
This should be on components level. So by default each component is using Fira Sans as a font.

Reply:
Do you mark those as resolved but have corresponding changes only locally?

Reply:
I pushed the changes to remote